src/api/booking/models/booking.entity.ts

```diff
@@ -5,14 +5,69 @@ import {
   Column,
   CreateDateColumn,
   UpdateDateColumn,
+  OneToOne,
+  JoinColumn,
 } from 'typeorm';
+import { User } from '../../user/models/user.entity';
+import { Car } from '../../car/models/car.entity';
+
+export enum bookingStatus {
+  PENDING = 'PENDING',
+  SUCCESS = 'SUCCESS',
+  FAIL = 'FAIL',
+}
+export enum pickUpStatus {
+  PENDING = 'PENDING',
+  PICKUP = 'PICKUP',
+  RETURNED = 'RETURNED',
+}
 
 @Entity()
 export class Booking {
   @PrimaryGeneratedColumn('uuid')
   @ApiProperty()
   public id!: string;
 
+  @OneToOne(() => User)
+  @JoinColumn()
+  user: User;
+
+  @OneToOne(() => Car)
+  @JoinColumn()
+  car: Car;
```

Comment:
Please create two join columns for this (like userId and carId) and put those in the @JoinColumn tag. This will make it a bit easier when you update the joined entities